prow/manifests/overlays/metal3/config.yaml

```
@@ -1947,6 +1952,11 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  # name: {job_prefix}-bml-integration-test-{image_os}
+  - name: metal3-bml-integration-tests-centos
+    agent: jenkins
+    always_run: false
+    optional: false
```

```suggestion
    optional: true
```
We don't want to make BML required test.

---

done